exonum/src/runtime/rust/mod.rs

```diff
@@ -121,12 +137,15 @@ impl RustRuntime {
             .expect("Method called before Rust runtime is initialized")
     }
 
+    /// Adds a new service factory to the list of available artifacts.
```

Comment:
This doc comment can lead to a conclusion that "service factory" is an "artifact", which is a bit confusing.